packages/ui/src/components/date-range-picker.tsx

```
@@ -0,0 +1,195 @@
+                  </Label>
+                  {dateRangePresets?.length &&
+                    dateRangePresets?.map((item) => (
+                      <Button
+                        variant="ghost"
+                        size="sm"
+                        className="w-full justify-start px-2 text-muted-foreground/80 hover:text-white hover:bg-accent-foreground"
+                        onClick={() => {
+                          setDate(item?.duration || undefined);
+                          setMonth(item?.duration?.to || today);
+                        }}
+                      >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)